### PR TITLE
Hand tracking module and landmarks export added

### PR DESCRIPTION
**Pull request by nata-vito**
Add the files responsible for hand detection and mapping.

### CODE REVIEW

File: hand_tracking.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            for handLms in self.results.multi_hand_landmarks:
+                if draw: self.mpDraw.draw_landmarks(img, handLms, self.mpHands.HAND_CONNECTIONS)
+
+        return img
+
+    def findPosition(self, img, handNo=0, draw=True):
+        lmList = []
+
+        if self.results.multi_hand_landmarks:
+            myHand = self.results.multi_hand_landmarks[handNo]
```
Comment (RodrigoRVSN) [inline hand_tracking.py:29]: what `handNo` means?

File: hand_tracking.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+            # Hand Labeling
+            for id, hand_handedness in enumerate(self.results.multi_handedness):
+                self.label = hand_handedness.classification[0].label
+
+            for id, lm in enumerate(myHand.landmark):
+                h, w, c = img.shape
+                cx, cy = int(lm.x * w), int(lm.y * h)
+                lmList.append([id, cx, cy])
```
Comment (RodrigoRVSN) [inline hand_tracking.py:38]: what this vars mean?
id, lm, h, w, c, cx, cy?
Could we rename this vars? Like, h = height

**Review by RodrigoRVSN — CHANGES_REQUESTED** [review]

File: hand_tracking.py (hunk 1 above)
Comment (nata-vito) [reply hand_tracking.py:29]: It means hands number, to indicate how many hands can be detect

File: hand_tracking.py (hunk 2 above)
Comment (nata-vito) [reply hand_tracking.py:38]: All right, this is just the first version. I'll uptade with new features and corrections.

id - > landmark identifier 
lm -> hand landmarks
h -> height
c -> image shape
cx -> landmark position in X
cy -> landmark position in Y

Comment (nata-vito) [reply hand_tracking.py:38]: References https://google.github.io/mediapipe/solutions/hands.html

**Pull request merged**